src/components/partials/Book.js

```diff
@@ -0,0 +1,31 @@
+/*eslint-disable */
+
+
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { actionRemoveBook } from '../../redux/books/books';
+
+const Book = (props) => {
+  const { book } = props;
+  const dispatch = useDispatch();
+  console.log(book);
```

Comment:
[ Optional ] But, it's a good practice to not leave console logs in your code file 👍 

- [ ] Please remove all console logs from your codebase

Reply:
Thanks for pointing this out.